api/src/movies/movies.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
Just a suggestion, it can be done now or in a follow-up PR: maybe we can add some messages after lists/movies are created/updated/deleted, very simple stuff for `prod` and maybe more detail in `dev`.

Reply:
See #45 👍